sstables/types.hh

```diff
@@ -529,6 +529,7 @@ enum class scylla_metadata_type : uint32_t {
     SSTableOrigin = 6,
     ScyllaBuildId = 7,
     ScyllaVersion = 8,
+    ClusteringPositionMetadata = 9,
```

Comment:
"Metadata" in the name seems redundant, since the whole component iscscylla metadata.
How about `FirstLastClusteringPositions`?

Reply:
Agreed.